examples/client/Locomotion/src/Components/CardRow/index.tsx

```diff
@@ -88,19 +116,8 @@ export default (paymentMethod: any) => (
           )
           : (
             <>
-              <PaymentIcon type={paymentMethod.brand} />
-              {paymentMethod.selected
-                ? (
-                  <SvgIcon
-                    style={{
-                      position: 'absolute',
-                      right: 0,
-                      bottom: 5,
-                    }}
-                    Svg={selected}
-                  />
-                )
-                : null}
+              {isCashPaymentMethod(paymentMethod) ? <SvgIcon Svg={cashIcon} width="40px" height="25px" /> : <PaymentIcon type={paymentMethod.brand} />}
+              {paymentMethod.selected ? (isCashPaymentMethod(paymentMethod) ? CashSelected : CreditCardSelected) : null }
```

Comment:
If it's the same component, just change the styles instead of 2 different components

Reply:
not more readable and short like this?